packages/@react-spectrum/s2/src/Tabs.tsx

```diff
@@ -198,35 +212,57 @@ const tablist = style({
   minWidth: 'min'
 });
 
+const tablistWrapper = style({
+  position: 'relative',
+  minWidth: 'min',
+  flexShrink: 0,
+  flexGrow: 0
+}, getAllowedOverrides());
+
 export function TabList<T extends object>(props: TabListProps<T>): ReactNode | null {
-  let {showTabs} = useContext(CollapseContext) ?? {};
+  let {showTabs, menuId, valueId, tabs, listRef, onSelectionChange, ariaLabel, ariaDescribedBy} = useContext(CollapseContext) ?? {};
+  let {density, orientation, labelBehavior} = useContext(InternalTabsContext);
 
   if (showTabs) {
     return <TabListInner {...props} />;
   }
-  return null;
+  
+  return (
```

Comment:
at some point, i know we [refactored the collapse behavior of Tabs](https://github.com/adobe/react-spectrum/pull/7600) due to an AXE accessibility error due to where we were rendering the menu but i've run AXE on this and nothing has come up

the reason as to why i moved it is because before, if a user chose to add a custom wrapper like this:
```
<div>
  <TabList />
  <div>
    <Button />
  </div>
</div>
```
what would happen is when the tabs collapsed, the wrapper would no longer wrap around both the menu + (hidden tabs) + buttons. so in order to ensure the wrapper actually wraps the expected item, i've moved the menu + hidden tabs into here